content/en/integrations/faq/powershell-command-to-install-azure-datadog-extension.md

```diff
@@ -8,16 +8,82 @@ Datadog provides an Azure extension to assist with Agent deployment on Azure ins
 * [Introducing Azure monitoring with one-click Datadog deployment][1]
 * [Azure integration documentation][2]
 
-An alternative to the GUI install is via Powershell.
-To run the Datadog Agent in your Azure instances as an extension, use the following syntax:
+An alternative to the GUI install is via command line.
+To run the Datadog Agent in your Azure instances as an extension, use the following syntaxes:
+
+{{< tabs >}}
+{{% tab "Windows" %}}
+
+```powershell
+Set-AzureVMExtension -Name "DatadogAgent" -Publisher "Datadog.Agent" -Type "DatadogWindowsAgent" -TypeHandlerVersion "5.0" -Settings @{"site" = "datadoghq.com"; "agentVersion" = "latest"} -ProtectedSettings @{"api_key" = "$(DatadogApiKey)"} -DisableAutoUpgradeMinorVersion
+```
+More information on the syntax to set Azure instance extensions can be found in the (Azure Extension Set-AzureVMExtension documentation)[3].
+
+The Azure Extension supports two type of settings: normal settings and protected settings.
```

Comment:
I think it would be helpful to link to the Microsoft docs for this, to make it more clear that normal vs protected settings are Microsoft Azure concept not a Datadog concept, and also so people can go to the Microsoft docs if they have questions about the level/method of protection.
https://learn.microsoft.com/en-us/azure/virtual-machines/extensions/features-windows#help-secure-vm-extension-data